tests/samples/gomod_replace_library/bar.go

```diff
@@ -2,6 +2,6 @@ package foo
 
 import "fmt"
 
-func Bar() {
```

Comment:
Bar should be exported, another project depends on this.
We need to remove tests/samples from golint, it may contain some 'wrong' code.